cmd/template/beegoRoutes.go

```diff
@@ -0,0 +1,29 @@
+package template
+
+type BeegoTemplates struct{}
+
+func (b BeegoTemplates) Main() []byte {
+	return MainTemplate()
+}
+func (g BeegoTemplates) Server() []byte {
+	return MakeHTTPServer()
+}
+func (g BeegoTemplates) Routes() []byte {
+	return MakeBeegoRoutes()
+}
+
+func MakeBeegoRoutes() []byte {
+	return []byte(`package main
+
+import (
+	"github.com/beego/beego/v2/server/web"
+	"fmt"
+)
+
+func main() {
+	fmt.println("Running at localhost:8080")
+	web.Run()
+}
+
+`)
+}
```

Comment:
i dont think this is going to work.
This looks like it is actually spinning up a server instance,
and not actually declaring any routes

Reply:
Yeah my mistake I misunderstood the task